pom.xml

```diff
@@ -60,7 +61,7 @@
     <junit.version>4.11</junit.version>
     <hamcrest.core.version>1.3</hamcrest.core.version>
     <maven.embedder.version>3.5.3</maven.embedder.version>
-    <curator.test.version>2.7.1</curator.test.version>
+    <curator.version>2.7.1</curator.version>
```

Comment:
Did you create a ticket with the a description for the issues of update onto curator 2.12.0 and 4.0.1 versions?
Could you post here the link for it?